bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js

```diff
@@ -198,8 +198,14 @@ pimcore.element.selector.asset = Class.create(pimcore.element.selector.abstract,
                 {text: t("filename"), width: 200, sortable: false, dataIndex: 'filename', hidden: true, renderer: Ext.util.Format.htmlEncode},
                 {text: t("preview"), width: 150, sortable: false, dataIndex: 'subtype',
                     renderer: function (value, metaData, record, rowIndex, colIndex, store) {
-                        if(record.data.subtype == "image") {
-                            var route = 'pimcore_admin_asset_getimagethumbnail';
+                        if(record.data.subtype == "image" || record.data.subtype == "video") {
```

Comment:
please fix it for documents as well. thanks!
https://github.com/pimcore/pimcore/blob/ea2f5884ec79540abd9093bb6b1f7c0832cfc010/bundles/AdminBundle/Controller/Admin/AssetController.php#L805-L806

Reply:
New code should support documents as well